sqlgen/mysql.go

```diff
@@ -80,6 +81,12 @@ func (q *SelectQuery) ToSQL() (string, []interface{}) {
 	buffer.WriteString(" FROM ")
 	buffer.WriteString(q.Table)
 
+	if len(q.Options.ForceIndex) > 0 {
+		buffer.WriteString(fmt.Sprintf(" FORCE INDEX(%s)", strings.Join(q.Options.ForceIndex, ",")))
```

Comment:
The convention of the library is to use 

```
buffer.WriteString(" <MySQL directive ")
buffer.WriteString(<table / input>)
```

I wonder if we want to avoid using `fmt.Sprintf`? I'm not sure why (maybe something to do with allocations). But would be nice to follow the convention of the library.

Reply:
Updated. Hopefully `strings.Join()` is acceptable. If not, let me know.